### ISSUE TITLE
Shinobido 2 Revenge of Zen (sound lost issue)

### ISSUE THREAD

**Issue by kuroyashizu** [Vita3K/Vita3K-Android#580]
So, this minor issue yet an annoying one hasn't been fixed yet, even after all the updates. Please, I need this problem solved for the next update. Every time I play the game for about 15-30 minutes, the sound effect suddenly lost during the gameplay, leaving only the ambience sound/backsound themes. Which is not fun at all. So please, fix this sound effect issue

**Comment by kuroyashizu** [comment]
I tried changing the sound setting on the emulator anyway, but to no avail. Like the SDL and Cubeb thing. The issue remains still

**Comment by Apolinariojason** [comment]
Hilow plss fix the sound bug ., shinobido 2
Sound lost....plzzzzzz

**Comment by kuroyashizu** [comment]
> Hilow plss fix the sound bug ., shinobido 2 Sound lost....plzzzzzz

That's what I'm asking to the developer of this emulator. I just hope they finally get the message

**Comment by xpufx** [comment]
Can you guys go into the options menu in this game? Not sound related but it's kicking me out right away.

**Comment by kuroyashizu** [comment]
> Can you guys go into the options menu in this game? Not sound related but it's kicking me out right away.

Have you tried that yourself?

**Comment by Coffeline** [comment]
Any updates on this? I've been regularly checking to see if the issue is fixed with each new update but still no luck :/
I've tested both USA and EUROPE versions of the game to see if it'd make a difference, but also nothing. In addition to that, I've also tried to play with and without updating the game to 1.02 but it seemed to have no effect either

**Comment by kuroyashizu** [comment]
> Any updates on this? I've been regularly checking to see if the issue is fixed with each new update but still no luck :/ I've tested both USA and EUROPE versions of the game to see if it'd make a difference, but also nothing. In addition to that, I've also tried to play with and without updating the game to 1.02 but it seemed to have no effect either

Yeah, there has been no updates, sadly. I've been dying waiting for so long for this issue to be fixed in the new update release. The sound effect still keeps getting lost after 30 minutes of playing. It's annoying. I love this game soo much, but this SFX issue hinders me from enjoying the game 😢

**Comment by DenisNinja** [comment]
[00:01:15.116] |W| [export_sceVideodecQueryMemSize::<lambda_1>::operator ()]: Stubbed sceVideodecQueryMemSize import called. (fake size)
[00:01:15.118] |W| [export_sceCodecEngineAllocMemoryFromUnmapMemBlock::<lambda_1>::operator ()]: Stubbed sceCodecEngineAllocMemoryFromUnmapMemBlock import called. (fake vaddr)
[00:01:15.122] |W| [export_sceAudiodecGetContextSize::<lambda_1>::operator ()]: Stubbed sceAudiodecGetContextSize import called. (fake size)
[00:01:22.275] |W| [export_sceCodecEngineFreeMemoryFromUnmapMemBlock::<lambda_1>::operator ()]: Stubbed sceCodecEngineFreeMemoryFromUnmapMemBlock import called. (always return success)
[00:01:22.276] |W| [export_sceKernelCloseMutex::<lambda_1>::operator ()]: Unimplemented sceKernelCloseMutex import called.

They did the hardest part, but no one solves the easiest part to fix in this emulator.

**Comment by leonaheidern464-source** [comment]
> [00:01:15.116] |W| [export_sceVideodecQueryMemSize::<lambda_1>::operator ()]: Stubbed sceVideodecQueryMemSize import called. (fake size) [00:01:15.118] |W| [export_sceCodecEngineAllocMemoryFromUnmapMemBlock::<lambda_1>::operator ()]: Stubbed sceCodecEngineAllocMemoryFromUnmapMemBlock import called. (fake vaddr) [00:01:15.122] |W| [export_sceAudiodecGetContextSize::<lambda_1>::operator ()]: Stubbed sceAudiodecGetContextSize import called. (fake size) [00:01:22.275] |W| [export_sceCodecEngineFreeMemoryFromUnmapMemBlock::<lambda_1>::operator ()]: Stubbed sceCodecEngineFreeMemoryFromUnmapMemBlock import called. (always return success) [00:01:22.276] |W| [export_sceKernelCloseMutex::<lambda_1>::operator ()]: Unimplemented sceKernelCloseMutex import called.
> 
> They did the hardest part, but no one solves the easiest part to fix in this emulator.

So you're saying it's almost impossible to fix?

**Comment by DenisNinja** [comment]
That's not what I meant, man—and just so you know, I'm not the developer of this emulator. I meant that this is the easiest thing for them to fix; I don't have the time to try and solve it myself.

**Comment by leonaheidern464-source** [comment]
> That's not what I meant, man—and just so you know, I'm not the developer of this emulator. I meant that this is the easiest thing for them to fix; I don't have the time to try and solve it myself.

I see... My apologies. My interpretation is off a little— well, yes. It's a shame, then. They refuse to acknowledge the easiest part to fix. I fail to understand why they chose the hardest, but not bothering to look at easier one. I love Shinobido 2 so much
